### PR TITLE
Arc 2643 audit log table entry

### PR DESCRIPTION
**Pull request by kamaksheeAtl**
**What's in this PR?**
set poco for new gh4j basic prod test

### CODE REVIEW

**Comment by yvonnevictoria** [comment]
@kamaksheeAtl Can you add a description to this PR please? :)

**Review by krazziekay — APPROVED** [review]

File: etc/poco/bundle/extras-prod-test.json [hunk 1]
```diff
@@ -90,6 +110,16 @@
...
 		},
+		{
+			"name": "Allow Prod Basic Check Critical Pollinator Test to call other admin endpoints",
+			"path": "/api/jira/13453453/verify",
+			"method": "DELETE",
+			"mechanism": "asap",
+			"principals": [
+				"pollinator-check/d4f03d07-12fe-4a69-9d68-c1841066772e"
+			],
+			"allowed": true
```
Comment (gxueatlassian) [inline etc/poco/bundle/extras-prod-test.json:121]: This should be false

File: etc/poco/bundle/extras-prod-test.json [hunk 2]
```diff
@@ -40,6 +40,16 @@
...
 		},
+		{
+			"name": "Allow Prod Basic Check Critical Pollinator Test to call Get Audit Log endpoints",
+			"path": "/api/audit-log/subscription/255625",
+			"method": "GET",
+			"mechanism": "asap",
+			"principals": [
+				"pollinator-check/d4f03d07-12fe-4a69-9d68-c1841066772e"
+			],
+			"allowed": false
```
Comment (gxueatlassian) [inline etc/poco/bundle/extras-prod-test.json:51]: You want this to be true to allow it.

File: etc/poco/bundle/extras-prod-test.json (hunk 1 above)
Comment (kamaksheeAtl) [reply etc/poco/bundle/extras-prod-test.json]: Done :)

**Review by gxueatlassian — APPROVED** [review]

**Pull request merged**